<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorWindow</class>
 <widget class="QMainWindow" name="CalculatorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CalculatorWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Shape:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>rectangle</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>triangle</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>circle</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Data</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="labelValue1">
             <property name="text">
              <string>Side a:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit1"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelValue2">
             <property name="text">
              <string>Side b:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit2"/>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="calculateButton">
             <property name="text">
              <string>Calculate</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>357</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="manuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionPolish"/>
    <addaction name="actionEnglish"/>
   </widget>
   <addaction name="manuLanguage"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPolish">
   <property name="text">
    <string>Polski</string>
   </property>
  </action>
  <action name="actionEnglish">
   <property name="text">
    <string>English</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
